### PR TITLE
Register and VAM Adjustments

### PR DESCRIPTION
**Pull request by biellamariscotes**
### Overview
- This PR refactors the VAM to the current flow. Also adds a note in the Register.

### Details
- Adds Student ID and full name details in Voter Details and Validate Voter:
image
image

- Refactors the table, replaced email with full names. Also implemented validation in the searching.

https://github.com/user-attachments/assets/dbaf8bf5-3ca0-4863-87ae-7c25b93a4599

### CODE REVIEW

File: src/manage-voters.php
Comment (Carl-Tabuso) [inline src/manage-voters.php:1]: Do change the jquery version to 3.7.1 so we won't have conflicting versions with other page files.

```html
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Also, add a defer attribute in the script tag of loader script.

```html
<script src="scripts/loader.js"></script>
```

or just simply move the script within the body—where all your external scripts are placed.

To make sure that the preloader is prioritized before your page renders, you can use the `preload` value for `rel` attribute. Kind of similar to this implementation.

```html
  <!-- Preloader Stylesheet and Image -->
  <link rel="preload" href="images/resc/ivote-icon.webp" as="image">
  <link rel="preload" href="styles/loader.css" as="style" />
  <link rel="stylesheet" href="styles/loader.css" />
```

**Review by Carl-Tabuso — CHANGES_REQUESTED** [review]
### Cosmetics

- Can you place the search icon inside the search input bar.
- You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).

image

- Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.

**Comment by biellamariscotes** [comment]
> ### Cosmetics
> * Can you place the search icon inside the search input bar.
> * You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).
> 
> image
> 
> * Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.

image

I have implemented the requested changes. The full name remains the first column since it serves as a hyperlink to the student's information. Using the full name is more appropriate for this redirection than the student ID, in my opinion.

File: src/manage-voters.php
Comment (biellamariscotes) [reply src/manage-voters.php:1]: Settled in commit [8611c64](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/pull/299/commits/8611c6492da59715d7ea1a1d450fafc9847a90d0).

**Comment by biellamariscotes** [comment]
COMMIT [804e231](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/pull/299/commits/804e231400e92c788431b38d9e01faa2d65d7e64)

Also removed the Add Voters, as well as the hiding of the pending registrations in SCO.

Excuse the data in the table, it's not updated.

image

File: src/manage-voters.php [hunk 1]
```diff
@@ -67,166 +68,168 @@ class="hide-text">MANAGE USERS</span>
...
+																	<th
+																		class="col-md-6 tl-right text-center fs-7 fw-bold spacing-5">
+																		<i data-feather="calendar"
+																			class="feather-xs im-cust"></i>Date Registered
+																	</th>
+																</tr>
+															</thead>
+															<tbody>
+																<?php while ($row = $to_verify_tbl->fetch_assoc()) { ?>
+																	<!-- Generated in table-funcs.js -->
```
Comment (Carl-Tabuso) [inline src/manage-voters.php]: This is what "Pending Registration" table looks like on my end.

image

**Comment by biellamariscotes** [comment]
@Carl-Tabuso fixed na

**Review by Carl-Tabuso — COMMENTED** [review]
You seem to have forgotten the Email Address header. Change this to Full Name and then we proceed with the merging.

image

**Comment by biellamariscotes** [comment]
> > ### Cosmetics
> > 
> > * Can you place the search icon inside the search input bar.
> > * You forgot to change the table header from Email Address to Full Name(also applicable to Voters' Accounts table).
> > 
> > image
> > 
> > * Would also like to add that you should actually create a student number column, ideally the first column. This shall serve as the unique identifier for each student as per sir chris and ma'am emy.
> 
> image
> 
> I have implemented the requested changes. The full name remains the first column since it serves as a hyperlink to the student's information. Using the full name is more appropriate for this redirection than the student ID, in my opinion.

image

Reverted back the missing codes that weren't included upon commit.

**Review by Carl-Tabuso — APPROVED** [review]

**Pull request merged**